public/components/common/query_utils/index.ts

```diff
@@ -219,6 +220,14 @@ export const getIndexPatternFromRawQuery = (query: string): string => {
   return getPromQLIndex(query) || getPPLIndex(query);
 };
 
+export const getDescribeQueryIndexFromRawQuery = (query: string): string => {
+  const matches = query.match(PPL_DESCRIBE_INDEX_REGEX);
+  if (matches) {
+    return matches[2];
+  }
+  return '';
```

Comment:
I'd recommend returning `undefined` explicitly instead of returning empty string. (It would enforce the check of `undefined`)

Reply:
To try and make this change with the least disruption possible, I followed the current method that an index is retrieved from the query here: https://github.com/opensearch-project/dashboards-observability/blob/411f33a154264b391df8dff751735dd51ebde67a/public/components/common/query_utils/index.ts#L210, which returns an empty string. Later on in the code the index as an empty string is used for some other purposes, and because of the legacy nature of this code, I decided to stick with whatever is done.
However, after looking through the code again, it seems like the final method I settled on for the describe query doesn't directly pass the parsed index anywhere else, meaning I can write it in as undefined. 
TLDR: Making this change now.